docs/docs/reference/release-notes/migrating-from-v2-to-v3.md

```diff
@@ -83,6 +83,28 @@ warning Plugin gatsby-plugin-acme is not compatible with your gatsby version 3.0
 
 This is because the plugin needs to set its `peerDependencies` to the new version of Gatsby (see section [for plugin maintainers](#for-plugin-maintainers)). While this might indicate that the plugin has incompatibilities, in most cases they should continue to work. Please look for already opened issues or PRs on the plugin's repository to see the status. If you don't see any, help the maintainers by opening an issue or PR yourself! :)
 
+#### Handling dependencies for plugins that are not yet updated
+
+ℹ️ If you run into the scenarios listed below, you will need to leverage yarn resolutions until the plugin authors upgrade the plugins they maintain.
```

Comment:
Note that I persisted the emoji sign-posts that @pragmaticpat had in the draft. That said, I don't think we have any style guide nor conventions here.

Reply:
I dropped it, as we don't use emojis this way